cves/kernel/CVE-2020-12770.yml

```diff
@@ -404,8 +435,11 @@ lessons:
     applies:
     note:
   security_by_obscurity:
-    applies:
-    note:
+    applies: true
+    note: |
+      The vulnerability could have been in the file since it was created. The file has had edits, the place 
```

Comment:
Were you able to find the commit that introduced the vulnerability into this file?

Reply:
I went all the way back to the first commit for that file and the vulnerability was there. I assumed that the vulnerability was in the system for a while.